doc/user/using-java10.md

```diff
@@ -1,20 +1,20 @@
-# Using TruffleRuby with JDK 9
+# Using TruffleRuby with JDK 10
 
 TruffleRuby is designed to be run with a JVM that has the Graal compiler. The
 easiest way to do this is to use the GraalVM, which includes JDK 8, the Graal
-compiler, and TruffleRuby, all in one package.
+compiler, TruffleRuby, and more, all in one package.
 
-It is also possible to run on Java 9, but we do not support this and at the
+It is also possible to run on Java 10, but we do not support this and at the
 moment we only document it for interest.
 
-To run on Java 9, Graal needs to be built with Java 9 and there is no binary
-distribution of GraalVM based on Java 9 so you will need to
-[build Graal yourself](../contributor/building-graal.md), using Java 9 as the
-JDK. You should also build TruffleRuby with the same JDK.
+To run on Java 10, Graal needs to be built with Java 10. As there is no binary
+distribution of GraalVM based on Java 10 you will need to [build Graal
+yourself](../contributor/building-graal.md), using Java 10 as the JDK (use the
+`install graal --no-jvmci` option to tell it to use your system Java 10). You
```

Comment:
Should it be `jt install graal --no-jvmci` ?

Reply:
Fixing.